### PR TITLE
throttle connection using a pool of 50 threads

### PR DESCRIPTION
**Pull request by ir73**

### CODE REVIEW

**Comment by admin-vizcentric** [comment]
There were errors, for info, please see...

**Comment by admin-vizcentric** [comment]
There were errors, for info, please see...

**Comment by admin-vizcentric** [comment]
There were no errors, go have a cup of coffee...

**Comment by admin-vizcentric** [comment]
There were no errors, go have a cup of coffee...

**Comment by admin-vizcentric** [comment]
There were no errors, go have a cup of coffee...

**Comment by admin-vizcentric** [comment]
There were no errors, go have a cup of coffee...

File: src/main/resources/config/application.yml [hunk 1]
```diff
@@ -1,3 +1,6 @@
+query-management:
```
[the remark this answers is not in the document]
Comment (ir73) [reply src/main/resources/config/application.yml:1]: I removed this key for now

**Pull request merged**